src/routes/user.routes.js

```diff
@@ -0,0 +1,31 @@
+import express from 'express';
+import { catchError } from '../untils/catchError.js';
+import { userController } from '../controllers/user.controller.js';
+import { authMiddleware } from '../middlewares/authMiddleware.js';
+
+export const userRouter = new express.Router();
+
+userRouter.get('/', authMiddleware, catchError(userController.getAllActivated));
+
+userRouter.post(
+  '/changeName/:userName',
```

Comment:
Ensure that the `userName` parameter is validated and sanitized to prevent potential security issues such as injection attacks.